.github/workflows/font.yml

```diff
@@ -2,6 +2,8 @@ name: Build Lucide
 on:
   push:
     branches: [ master ]
+  pull_request:
+    branches: [ master ]
```

Comment:
is there a reason why you want to build the font when a pull-request is opened?
I changed it, because it only build when a PR is merged to master.